web_external/model.js

```diff
@@ -0,0 +1,14 @@
+isic.Model = girder.Model.extend({
+    urlRoot: function () {
+        return this.resourceName;
+    }
+});
+
+Backbone.ajax = function (options) {
+    // girder.restRequest expects a "path" option, and will set "url" internally
+    var params = _.clone(options);
+    params.path = params.url;
+    delete params.url;
+
+    return girder.restRequest(params);
```

Comment:
This gives me `Maximum call stack size exceeded` because `girder.restRequest()` calls `Backbone.ajax()`.